frontend/packages/console-app/src/components/quick-starts/QuickStartCatalogPage.tsx

```diff
@@ -1,31 +1,31 @@
 import * as React from 'react';
-import { Text } from '@patternfly/react-core';
+import { QuickStartCatalogPage as PfQuickStartCatalogPage } from '@patternfly/quickstarts';
 import { Helmet } from 'react-helmet';
 import { useTranslation } from 'react-i18next';
 import { LoadingBox } from '@console/internal/components/utils';
-import QuickStartCatalog from './catalog/QuickStartCatalog';
 import QuickStartsLoader from './loader/QuickStartsLoader';
 
 const QuickStartCatalogPage: React.FC = () => {
   const { t } = useTranslation();
   return (
     <>
       <Helmet>
-        <title>{t('quickstart~Quick Starts')}</title>
+        <title>{t('console-app~Quick Starts')}</title>
```

Comment:
this should be 
```suggestion
        <title>{t('quickstart~Quick Starts')}</title>
```

since we have separate locales for quick starts

Reply:
ok, so there are 2 reasons why I moved those 2 translations into console
- Those keys are not used in the quick start library itself, they are only used by OpenShift to add title and subtitle to the catalog
- Because there are no more `quickstart~` keys used in the app (I just load the whole quickstart resource bundle and feed it to the library via context value), if there is a `quickstart~` key, the i18n script will actually try to update the `quickstart.json` file with only the keys it found, which there would only be 2 then, so the CI job would fail.